app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.job_id = params[:job_id]
+    @comment.save
```

Comment:
what happens if the comment doesn't save? if the user enters invalid data, we should show them a nice error message. be sure to handle this case!